lib/APIPlugin.js

```diff
@@ -30,8 +30,18 @@ const REPLACEMENT_TYPES = {
 	__webpack_nonce__: "string"
 };
 /* eslint-enable camelcase */
+/** @typedef {import("./Parser")} Parser */
```

Comment:
Don't change other file please

Reply:
Which file?

Reply:
aaa I see what you mean, sh...... ok, sorry will fix this